examples/3d-tiles/tile-3d-layer/tile-3d-layer.js

```diff
@@ -345,9 +259,39 @@ export default class Tile3DLayer extends CompositeLayer {
     });
   }
 
+  // TODO - delete when bug fixed
+  _resolveTransformProps(tileHeader) {
```

Comment:
this was moved to `modules/3d-tiles/src/tileset/helpers/transform-utils`.

Reply:
Yes, I intentionally copied it back, because there is a regression in the new code in `transform-utils` (the layer no longer shows all tiles when zooming in). I tried to debug it but couldn't easily find the issue. Restoring this code path makes it work again.